extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -1542,6 +1542,15 @@ public static Token fromAudience(String... audience) {
         @ConfigItem
         public Optional<Duration> age = Optional.empty();
 
+        /**
+         * Require that the token includes a `iat` (issued at) claim
+         *
+         * Set this property to `false` if your JWT token doesn't contain an `iat` claim,
+         * or you don't want to enable verification of it. This property will be ignored for the ID token verification.
```

Comment:
```suggestion
         * Note that ID token is always required to have an `iat` claim and therefore this property has no impact on the ID token verification process.
```

I'm suggesting a minor rewording, as well as dropping a point where a user may choose to ignore the fact that the iat is present